src/DependencyInjection/Configuration.php

```diff
@@ -24,11 +24,13 @@ public function getConfigTreeBuilder()
                 ->arrayNode('query')
                     ->children()
                         ->arrayNode('middlewares')
+                            ->info('chain named of middlewares')
```

Comment:
I'm sorry but I think that this description is not explicit enough.
Could you explain it in french eventually ? I could then suggest some english translation.

This suggestion goes along with all "->info()" calls 😄 

👍

Reply:
ok, je viendrais à coté de toi pour cela. je pense que ce sera plus simple pour échanger sur une bonne description pour chacune des descriptions de cette configuration